src/sign.builds

```diff
@@ -15,6 +15,11 @@
     <OutDir>$(BinDir)</OutDir>
   </PropertyGroup>
 
+  <PropertyGroup>
```

Comment:
I would suggest the following:
```
<SkipFileSigning Condition="''$(SkipFileSigning)' == '' and "'$(BuildArch)' == 'arm64'">true</SkipFileSigning>
```

And then change your conditions below to `'$(SkipFileSigning)' != 'true'`